### DISCUSSION TITLE
Q-dialog Full Width

### DISCUSSION THREAD

**Discussion by Divensky** [quasarframework/quasar#16869] [General - Components / Directives / etc]
Hi! Is it by design that Q-dialog in Full-Width mode is not really **full** width but has padding applied to it? 
```
.q-dialog__inner--minimized {
    padding: 24px;
} 
```
![full-width-quasar](https://github.com/quasarframework/quasar/assets/116442718/570e7d99-6ea3-4363-84b9-a296380d17da)

I was expecting my full-width component to take the entire width with no left-over background on the left and right. I will override the padding for the moment, but I am not sure it was meant to be.

**Comment by ibrainventures** [comment]
![grafik](https://github.com/quasarframework/quasar/assets/8803626/41cbddd8-1e58-46e9-b39a-ce489f9ba56c)

How does it look when you toggle the maximized bool on true?

**Reply by Divensky** [reply]
I did not use maximized in my project but on Quasar website it looks correct, i.e. it takes the entire screen with no padding on the sides, unlike the full-width on the screen shot above:
![image](https://github.com/quasarframework/quasar/assets/116442718/06bbc089-e774-49ce-9ab4-4a1793f3c762)
